h/accounts/views.py

```diff
@@ -348,7 +345,7 @@ def register(self):
             return {'form': self.form.render()}
 
         self._register(username=appstruct['username'],
-                       email=appstruct['email'],
+                       email_address=appstruct['email'],
```

Comment:
The field in the appstruct is called `email`, and the keyword argument to the user constructor is called `email`, so passing it through a thing called `email_address` just looks weird.

Perhaps we could call the module `h.mailer`?